contact-form/src/Schemas/signup-company.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+import student from '../Definitions/student';
+import company from '../Definitions/company';
+
+export const submitFunction = (formData, endpoint) => {
+  const headers = { 'Content-Type': 'application/json' };
+  const instance = axios.create({ headers });
+  const baseUrl = window.location.origin;
+
+  const template = document.createElement('div');
+  window.parent.scrollTo(0, 0);
+  window.scroll(0, 0);
+
+  instance.request({
+    method: 'post',
+    data: JSON.stringify(formData),
+    url: `${baseUrl}/wp-json/dationwoocommerce/v1/submit/${endpoint}`,
+  }).then(() => {
+    template.innerHTML = 'Inschrijving voldaan';
+    template.className = 'alert alert-success';
+
+    const placeHolder = document.getElementById('alertPlaceHolder');
+    placeHolder.append(template);
+    // Show success message and redirect to home after a short timeout
+    setTimeout(() => {
+      window.parent.location.replace(baseUrl);
```

Comment:
Why not a confirm modal or something else that requires user input before automatically redirecting